AmplifyPlugins/Analytics/AWSPinpointAnalyticsPluginIntegrationTests/Info.plist

```diff
@@ -13,7 +13,7 @@
 	<key>CFBundleName</key>
 	<string>$(PRODUCT_NAME)</string>
 	<key>CFBundlePackageType</key>
-	<string>BNDL</string>
+	<string>$(PRODUCT_BUNDLE_PACKAGE_TYPE)</string>
```

Comment:
Why did this change?

Reply:
the test target was created awhile ago so i'm not sure which type of target was selected, but it only allowed specifying "Target Application" to be the HostApp to execute the tests.  this was a problem because when we add a configuration file with the HostApp target (copy to bundle resources), it gets added to the HostApp's bundle but when the tests executed, it would not find the file from `Bundle.main`. I had to delete the target and recreate it with the unit test bundle, and this new Info.plist came with it. With the new target, it then allows specifying the "Host Application" which we still set to HostApp, and test execution worked as Bundle.main was able to read the configuration file into memory

Reply:
Thanks for the explanation. tl;dr: Frickin' Xcode, man.